daemon/kill.go

```diff
@@ -55,7 +55,7 @@ func (daemon *Daemon) ContainerKill(name string, sig uint64) error {
 // or not running, or if there is a problem returned from the
 // underlying kill command.
 func (daemon *Daemon) killWithSignal(container *container.Container, sig int) error {
-	logrus.Debugf("Sending %d to %s", sig, container.ID)
+	logrus.Debugf("Sending kill signal %d to container %s", sig, container.ID)
```

Comment:
@thaJeztah 
I prefer "kill signal".
As, no matter `docker stop` or `docker kill`, in the os level, there is always a kill signal, just different in signal number.